core/build/build.go

```diff
@@ -0,0 +1,94 @@
+package build
+
+import (
+	"context"
+	"runtime/debug"
+
+	"github.com/aerogear/charmil/core/color"
+	"github.com/aerogear/charmil/core/localize"
+	"github.com/aerogear/charmil/core/logging"
+	"github.com/google/go-github/github"
+)
+
+// Define public variables here which you wish to be configurable at build time
+var (
+	// Version is dynamically set by the toolchain or overridden by the Makefile.
+	Version = "dev"
+
+	// Language used, can be overridden by Makefile or CI
+	Language = "en"
+
+	// RepositoryOwner is the remote GitHub organization for the releases
+	RepositoryOwner = "redhat-developer"
```

Comment:
Please do not copy code to core

Reply:
yes, this has to be configurable by CLI. I already made a note for it.. will not push in main with this